src/dmd/mars.d

```diff
@@ -2376,3 +2395,16 @@ Modules createModules(ref Strings files, ref Strings libmodules)
     }
     return modules;
 }
+
+/*
+Checks whether the option pointer supplied is a request
+for the help page.
+
+Returns: `true` if `p` points to `?`, `h` or `help`, otherwise `false`.
+*/
+bool isHelpOption(const(char*) p)
+{
+    return strcmp(p, "?") == 0 ||
+           strcmp(p, "h") == 0 ||
+           strcmp(p, "help") == 0;
```

Comment:
On second thought, with this function I'm all in.

Reply:
Yeah, just accept all three. I know I try them in sequence (for random utilities) until one of them works.